### PR TITLE
1158 custom checkbox element implement

### PR DESCRIPTION
**Pull request by VeronikaRevjakina**
Implement CustomCheckbox https://getbootstrap.com/docs/4.3/components/forms/#checkboxes 
and test on it

### CODE REVIEW

**Comment by VeronikaRevjakina** [comment]
In BootstrapPage.page no element was deleted ( it is in the file in another place, i checked no conflicts with bootstrap branch)

**Comment by VeronikaRevjakina** [comment]
DO NOT MERGE
This change will AFFECT ALL TESTS and composite elements which use Checkbox inside, 
changes on locators required (from inpur to div) (jn BootstrapPage, inside composite),
needed fixes on tests where checkbox().label() is used to click() on elements-NavBar & CheckboxesAndRadios

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Checkbox.java [hunk 1]
```diff
@@ -1,41 +1,79 @@
...
-    implements HasLabel, SetValue, HasClick, HasCheck {
+        implements HasLabel, SetValue, HasClick, HasCheck {
+
+    @UI("input")
+    private UIElement input;
+
     // region Actions
     public void check(String value) {
-        if ("true".equalsIgnoreCase(value) || ("1").equals(value))
+        if ("true".equalsIgnoreCase(value) || ("1").equals(value)) {
```
Comment (AUlasen) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Checkbox.java]: It is not necessary to put "1" in parentheses

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Checkbox.java [hunk 2]
```diff
@@ -1,41 +1,79 @@
...
     // endregion
 
     // region Set and get value for Forms
     public void setValue(String value) {
         check(value);
     }
+
     public String getValue() {
-        return isSelected()+"";
+        return isSelected() + "";
```
Comment (AUlasen) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Checkbox.java]: It is better ro use String.valueOf() to convert boolean to String

**Pull request merged**